src/UI/NavigationContainer.elm

```diff
@@ -49,7 +49,7 @@ Example of usage:
 
 # Model & Update
 
-@docs Msg, State, stateInit, stateUpdate
+@docs Msg, State, stateInit, stateInitWithClosedMenu, stateUpdate
```

Comment:
:thinking: why not a proper `stateWithMenuOpen : Boolean -> State -> State`?

Reply:
because so far there's no need for a state that is forced open

Reply:
then, what about `stateInit |> stateWithMenuClosed` ?

Reply:
done